### PR TITLE
Adding index VTEX Shipping Network API

### PR DESCRIPTION
**Pull request by Isabella-Veloso**
#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [x] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [ ] No, but I am going to.

### CODE REVIEW

File: VTEX - VTEX Shipping Network API.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
...
...name-\/v-app_version-\/-account-\/-workspace-\/tracking)\r\n\r\n## Common parameters\r\n\r\n| **Parameter name** | **Description** | **Type** |\r\n| :---: | :--- | :--- |\r\n| `{{accountName}}` | Name of the VTEX account. Used as part of the URL. | Server variable. |\r\n| `{{environment}}` | Environment to use. Used as part of the URL. The default value is `vtexcommercestable`. | Server variable. |\r\n| `X-VTEX-API-AppKey` | Unique identifier of the [application key](https://developers.vtex.com/docs/guides/api-authentication-using-application-keys). | Authentication header. Must be used together with `X-VTEX-API-AppToken`. Not necessary when using `VtexIdclientAutCookie`. |\r\n| `X-VTEX-API-AppToken` | Secret token of the [application key](https://developers.vtex.com/docs/guides/api-authentication-using-application-keys). | Authentication header. Must be used together with X-VTEX-API-AppKey. Not necessary when using `VtexIdclientAutCookie`. |\r\n| `VtexIdclientAutCookie` | [User token](https://developers.vtex.com/docs/guides/api-authentication-using-user-tokens), valid for 24 hours. | Authentication header. Not necessary when using `X-VTEX-API-AppKey` and `X-VTEX-API-AppToken`. |",
```
Comment (julia-rabello) [inline VTEX - VTEX Shipping Network API.json:6]:
```suggestion
      "description": "[VTEX Shipping Network](https://vtex.com/br-pt/shipping-network/) is a solution that connects carriers and merchants for deliveries at better freight costs and smarter logistics operations.\r\n\r\n>ℹ️ **VTEX Shipping Network** is currently available exclusively in Brazil.\r\n\r\nThe tracking process has two main parts:\r\n- **Notification:** The [carrier](https://help.vtex.com/en/tutorial/carriers-on-vtex--7u9duMD5UQa2QQwukAWMcE) is notified that a package is being dispatched.\r\n- **Tracking:** The package being delivered by the carrier is tracked from the start of its journey to its final destination.\r\n\r\nTo integrate with the VTEX Shipping Network hub, carriers must develop [VTEX IO](https://developers.vtex.com/docs/guides/vtex-io-documentation-what-is-vtex-io) apps and associate them with specific endpoints. Once integrated with the hub, these carrier apps will be connected to other VTEX systems, ensuring seamless execution of the necessary steps for the tracking process.\r\n\r\n>ℹ️ For detailed instructions, see the [VTEX Shipping Network integration guide](https://developers.vtex.com/docs/guides/vtex-shipping-network).\r\n\r\n## VTEX Shipping Network API Index\r\n\r\n### Notification\r\n\r\n- `POST` [Notify carrier with app](https://developers.vtex.com/docs/api-reference/vtex-shipping-network-api#post-/-app_name-/v-app_version-/-account-/-workspace-/notify)\r\n\r\n### Tracking\r\n\r\n- `POST` [Tracking events with app](https://developers.vtex.com/docs/api-reference/vtex-shipping-network-api#post-/-app_name-/v-app_version-/-account-/-workspace-/tracking)\r\n\r\n## Common parameters\r\n\r\n| **Parameter name** | **Description** | **Type** |\r\n| --- | --- | --- |\r\n| `{{accountName}}` | Name of the VTEX account. Used as part of the URL. | Server variable. |\r\n| `{{environment}}` | Environment to use. Used as part of the URL. The default value is `vtexcommercestable`. | Server variable. |\r\n| `X-VTEX-API-AppKey` | Unique identifier of the [application key](https://developers.vtex.com/docs/guides/api-authentication-using-application-keys). | Authentication header. Must be used together with `X-VTEX-API-AppToken`. Not necessary when using `VtexIdclientAutCookie`. |\r\n| `X-VTEX-API-AppToken` | Secret token of the [application key](https://developers.vtex.com/docs/guides/api-authentication-using-application-keys). | Authentication header. Must be used together with X-VTEX-API-AppKey. Not necessary when using `VtexIdclientAutCookie`. |\r\n| `VtexIdclientAutCookie` | [User token](https://developers.vtex.com/docs/guides/api-authentication-using-user-tokens), valid for 24 hours. | Authentication header. Not necessary when using `X-VTEX-API-AppKey` and `X-VTEX-API-AppToken`. |",
```

Comment (julia-rabello) [inline VTEX - VTEX Shipping Network API.json:6]: The following change you made to the description seems to have switched the order of information.

> The carrier's apps have to integrate with the hub, which is connected to other VTEX systems. The carrier is required to develop [VTEX IO](https://developers.vtex.com/docs/guides/vtex-io-documentation-what-is-vtex-io) apps and associate them with specific endpoints.

In my opinion, it should be in this order: Carriers have to develop IO apps -> these IO apps will integrate with the hub, which is connected to other VTEX systems.

**Review by julia-rabello — CHANGES_REQUESTED** [review]

File: VTEX - VTEX Shipping Network API.json (hunk 1 above)
Comment (Isabella-Veloso) [reply VTEX - VTEX Shipping Network API.json]: Suggestion accepted!

**Comment by Isabella-Veloso** [comment]
@julia-rabello I have accepted your reviewed suggestion. I just removed the word "currently" from the callout below because it is an exclusive solution for Brazil and will not be scaled to other countries.

**VTEX Shipping Network** is available exclusively in Brazil.

**Review by julia-rabello — APPROVED** [review]

**Pull request merged**